spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -10,9 +10,16 @@ def install_engine
   def create_generic_work
     generate 'hyrax:work GenericWork'
   end
-  
+
   def create_atlas_work
     generate 'hyrax:work RareBooks/Atlas'
+    append_file 'config/initializers/inflections.rb' do
+      <<~HEREDOC
```

Comment:
@mjgiarlo for future people look at this code can you add a comment that this was done becuase it created a bad plural?